docs/source/array-api.rst

```diff
@@ -98,6 +98,7 @@ Top level user functions:
    greater
    greater_equal
    histogram
+   histogram2d
```

Comment:
Could you add a corresponding `.. autofunction::` entry for `histogram2d`?

Reply:
done :+1: